README.md

```diff
@@ -201,6 +201,34 @@ See [here](#arguments)
 ###### Successful message
 See [here](#successful-message)
 
+#### Option 4. Use Docker image
+
+##### Step 1. Create keys and `deposit_data-*.json`
```

Comment:
Should the docs tell the user to build the docker image first? (`make build_docker`)